lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/LessonBuilderAccessService.java

```diff
@@ -169,6 +170,18 @@ public void setSiteService(SiteService s) {
 		siteService = s;
 	}
 
+	private UserTimeService userTimeService;
+
+	public void setUserTimeService(UserTimeService s) {
+		userTimeService = s;
+	}
+
+	private UserDirectoryService userDirectoryService;
+
+	public void setUserDirectoryService(UserDirectoryService s) {
```

Comment:
could just use the @Setter annontation and avoid these methods!

Reply:
lombok.Setter

Reply:
Updated. I figured I would follow the existing pattern in the class but makes sense to just update as we go.